src/solution/server/routes.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
```

Comment:
Great routing file! you can also create another layer and have a specific router for items, as in the near future you may have another type of data you would like to maintain and this file will get longer and messy.
Fruits router - will return a router listening to these routes for example `/fruits` `/fruits/:id'